docs/data-backlog.md

```diff
@@ -0,0 +1,133 @@
+
+We've also proposed various "milestones" that will allow us to iteratively and sequentially build the data backlog in a way that supports the entire vision but starts with small and achievable steps. These milestones are proposed in order.
+
+## Milestone: YAML files for data sets and linter
+
+To start, we'll create a folder in the appropriate repository [is this `justice40-tool`?] that can house YAML files, one per data set. Each file will describe the characteristics of the data.
+
+The benefit of using a YAML file for this is that it's easy to subject changes to these files to peer review through the pull request process. This allows external collaborators from the open source community to submit suggested changes, which can be reviewed by the core CEJST team.
+
+We'll use a Python-based script to load all the files in the directory, and then run a schema validator to ensure all the files have valid entries.
+
+For schema validation, we propose using [Marshmallow](https://marshmallow.readthedocs.io/en/stable/). This is slightly heavier-weight than necessary, but since we're planning to use Marshmallow as a serialization library elsewhere in the project, it helps to keep things consistent.
+
+If there's an improper format in any of the files, the schema validator will throw an error.
+
+As part of this milestone, we will also set this up to run automatically with each commit to any branch as part of CI/CD.
+
+## Milestone: Post data in tabular format
+
+Add a script that runs the schema validator on all files and, if successful, posts the results in a tabular format. There are straightforward packages to post a Python dictionary / `pandas` dataframe to Google Sheets and/or Airtable. As part of this milestone, we will also set this up to run automatically with each commit to `main` as part of CI/CD.
+
+This will make it easier to filter the data to answer questions like, "which data sources are available at the census block group level".
+
+## Milestone: Tickets created for incorporating data sets
+
+For each data set that is being considered for inclusion soon in the tool, the project management team will create a ticket for "Incorporating ___ data set into the database", with a link to the data set detail document. This ticket will be created in the ticket tracking system used by the open source repository, which is ZenHub. This project management system will be public.
+
+At the initial launch, we are not planning for members of the open source community to be able to create tickets, but we would like to consider a process for members of the open source community creating tickets that can go through review by the CEJST team.
+
+This will help developers know what to work on next, and open source community members can also pick up tickets and work to integrate the data sets.
+
+## Milestone: Google forms integration
```

Comment:
It might make sense to start with this milestone? Once we have a backlog we can work on the tooling to integrate